pinot-core/src/main/java/org/apache/pinot/core/query/request/context/ExpressionContext.java

```diff
@@ -85,6 +89,29 @@ public void getColumns(Set<String> columns) {
     }
   }
 
+  /**
+   * Temporary helper method to help the migration from BrokerRequest to QueryContext.
+   */
+  public TransformExpressionTree toTransformExpressionTree() {
```

Comment:
Do we need a unit test for this? Until now we were using a different implementation to convert to TET. Now we have this one. May be as long as all the end to end query execution tests pass, we are good?

Reply:
Yes, and this is a temporary helper method which will be removed soon.